docs/campaign/quests/web3-evm.md

```diff
@@ -2,16 +2,32 @@
 
 Using the Web3.0 EVM Task, you can easily put conditions on a smart contract and verify onchain actions. Let's dive deep into how can we create a web 3.0 EVM task at AirLyft! 
 
-- Login to AirLyft and create an event. On this, head to the select Apps section and select Web 3.0 EVM block as shown in the screenshot below. 
+- Login to AirLyft and create an event. On this, head to the Quests tab and select Web 3.0 EVM block as shown in the screenshot below. 
 
 ![](../../images/web3evmMain.png)
 
 - At the very first step, enter the basic details about the EVM Task that you are creating. It is advisable to elaborate on the title and description so that participants have a much greater understanding of the task. For instance, let's suppose you want to check if a user holds a 10 KTE token or not.
 
 ![](../../images/web3Basic.png)
 
+- Then expand 'Task Condition and Recurrence' option to set the task frequency and task conditions. 
```

Comment:
This is okay, but we could have also created another page with "Task Conditions & Recurrence" and added a link to that page. Nut now let's not change this.